### PR TITLE
Feat: contacts page

### PR DESCRIPTION
**Pull request by nanjiangshu**
**Related issue(s) and PR(s)**  
This PR  relates to #859.

Implement the contact page for the new about page.

**Type of change**  

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other
 
**List of changes made**  

Implemented the contact page. The `to` email address will be filled in when clicking the link. 

**Screenshot of the fix**  

**Testing**  

**Further comments**  

**Checklist**  

- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have checked so that the same data as before is returned by the API

### CODE REVIEW

File: frontend/src/components/about/Contact.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
-  <about-layout title="License">
+  <about-layout title="Contacts">
```
Comment (mihai-sysbio) [inline frontend/src/components/about/Contact.vue:2]: I think I prefer one of these alternatives better:

```suggestion
  <about-layout title="Contact">
```

or 

```suggestion
  <about-layout title="Contact us">
```

File: frontend/src/components/about/Contact.vue [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     <template v-slot:contents>
       <p>
         For any questions, comments or suggestions please email us at
-        <a href="mailto:">contact [at] metabolicatlas [dot] org</a>
+        <a href="mailto:contact@metabolicatlas.org">contact [at] metabolicatlas [dot] org</a>
```
Comment (mihai-sysbio) [inline frontend/src/components/about/Contact.vue:6]: This was not added so at not to be scraped, but I think it's okay to have it 👍🏻

**Review by mihai-sysbio — APPROVED** [review]

File: frontend/src/components/about/Contact.vue (hunk 1 above)
Comment (nanjiangshu) [reply frontend/src/components/about/Contact.vue]: Changed in the commit f1c3273d

Besides, I've also added the description for how to report bugs and vulnerabilities as well as how to use management@metabolicatlas.org

File: frontend/src/components/about/Contact.vue (hunk 2 above)
Comment (nanjiangshu) [reply frontend/src/components/about/Contact.vue:7]: True. But it is also troublesome for the user to click the link with the `to` address unfilled.

**Pull request closed without merging**